themes/beaver/layouts/partials/page/header.html

```diff
@@ -0,0 +1,151 @@
+
+      {{ else if eq .Type "services" }}
+        {{ partial "header/single/services.html" . }}
+
+      {{ else if eq .Type "use-cases" }}
+        {{ partial "header/single/use-cases.html" . }}
+
+      {{ else if .IsHome }}
+        {{ partial "header/homepage.html" . }}
+
+      {{ else if eq .Params.slug "contact-us" }}
+        {{ partial "header/contact-us.html" . }}
+
+      {{ else if eq .Params.slug "about-us" }}
+        {{ partial "header/about-us.html" . }}
+
+      {{ else if (strings.HasPrefix $url "/blog/") }}
+        {{ partial "header/blog.html" . }}
+      {{ end }}
+
+      {{ if eq .Params.slug "careers" }}
+        {{ partial "header/careers.html" . }}
+
+      {{ else if eq .Type "careers" }}
+        {{ partial "header/single/careers.html" . }}
+
+      {{ else if eq .Type "clients" }}
+        {{ partial "header/single/clients.html" . }}
+      {{ end }}
+
+      {{ if eq .Type "tags" }}
+      {{ partial "header/blog.html" . }}
+      {{ end }}
+
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-migrate/3.4.1/jquery-migrate.min.js" id="jquery-migrate-js"></script>
+      <link rel="icon" href="{{ absURL "wp-content/uploads/2022/09/cropped-jet-thoughts-logo-32x32.png" }}" sizes="32x32" />
+      <link rel="icon" href="{{ absURL "wp-content/uploads/2022/09/cropped-jet-thoughts-logo-192x192.png" }}" sizes="192x192" />
+      <link rel="apple-touch-icon" href="{{ absURL "wp-content/uploads/2022/09/cropped-jet-thoughts-logo-180x180.png" }}" />
+      <meta name="msapplication-TileImage" content="{{ absURL "wp-content/uploads/2022/09/cropped-jet-thoughts-logo-270x270.png" }}" />
+
+       <!-- Google tag (gtag.js) -->
+       <script async src="https://www.googletagmanager.com/gtag/js?id=G-T1N27FC30F"></script>
+       <script>
+           window.dataLayer = window.dataLayer || [];
+           function gtag(){dataLayer.push(arguments);}
+           gtag('js', new Date());
+
+           gtag('config', 'G-T1N27FC30F');
+       </script>
+
+      {{ if .IsHome }}
+        <link rel="prefetch" href="/services/" as="document" >
+        <link rel="prefetch" href="/use-cases/" as="document">
+        <link rel="prefetch" href="/about-us/" as="document">
+        <link rel="prefetch" href="/clients/" as="document">
+        <link rel="prefetch" href="/careers/" as="document">
+      {{ end }}
+      <link rel='stylesheet' id='fl-builder-layout-3059-css' href='{{ absURL "assets/css/style.css" }}' media='all' />
+   </head>
+
+    <body class="privacy-policy page-template-default page page-id-3059 fl-builder fl-theme-builder-header fl-theme-builder-header-header fl-theme-builder-footer fl-theme-builder-footer-footer fl-framework-base-4 fl-preset-default fl-full-width fl-search-active" itemscope="itemscope" itemtype="https://schema.org/WebPage">
+      <div class="fl-page">
+
+        {{ partial "page/top-panel.html" . }}

```

Comment:
instead of adding new files, please update `baseof.html` and replace appropriate `header` tag with `{{ partial "page/top-panel.html" . }}`